assets/stylesheets/components/_global-header.scss

```diff
@@ -4,7 +4,11 @@
 
 .global-header--inner {
   @extend %content-block;
-  padding: 24px;
+  padding: ($baseline-grid-unit * 4);
```

Comment:
In fact, it's actually already being looked at as part of #27 (https://github.com/nhsalpha/betahealth/pull/27/files#diff-258a86056b2ecc311b8da35def260643R7)